src/main/resources/assets/js/search.js

```diff
@@ -0,0 +1,43 @@
+window.onload = function() {
```

Comment:
Note to self: https://www.w3schools.com/jsref/event_onpageshow.asp might be a better idea than `onload` to prevent quirks with navigate back.